json_to_csv.py

```diff
@@ -0,0 +1,46 @@
+import os
+import json
+import csv
+
+
+def fun(in_dir,out_dir,**d):

```

Comment:
function name must be readable. And remove param `d` if unused

Reply:
Ok